dotcom-rendering/src/web/components/DynamicFast.tsx

```diff
@@ -68,6 +68,7 @@ const Primaries = ({
 						>
 							<FrontCard
 								trail={card}
+								starRating={card.starRating}
```

Comment:
Do we need these changes?

I think `FrontCard` will pass this `starRating` property through by default